docs/en/mods/armaforces_medical.md

```diff
@@ -0,0 +1,93 @@
+
+### Dressings efficiency table
+
+`++` = Excellent | `+` = Increased | None = Standard
+
+| Wound\Dressing| Field Dressing |  Elastic  |  Packing | QuikClot |
+|:-----------:|:--------------:|:---------:|:--------:|:--------:|
+|   Abrasion  |       ++       |           |          |          |
+|  Avulsion   |                |     ++    |          |          |
+|  Contusion  |                |     ++    |          |          |
+|    Crush    |                |     ++    |          |          |
+|     Cut     |       ++       |           |          |          |
+|  Laceration |       ++       |           |          |          |
+|   Velocity  |        +       |      +    |    ++    |    ++    |
+|   Puncture  |        +       |      +    |    ++    |    ++    |
+
+The table below shows additional parameters worth taking into consideration when applying specific dressing. 
+
+- timeframe throughout which it is possible for bleeding to start again,
+- chance of bleeding occurring in aforementioned timeframe (if bleeding doesn't happen under these terms, it won't happen at all),
+- how many wounds of specific size you can dress using specific dressing
+
+| Efficiency            |      | +    | ++    |
+|-----------------------|:----:|:----:|:-----:|
+| Bleeding timeframe    | 5-10 | 6-20 | 10-30 |
+| Bleeding chance       | 0.4  | 0.3  | 0.1   |
+| Large wounds          | 2    | 4    | 8     |
+| Medium wounds         | 3    | 5    | 9     |
+| Minor wounds          | 4    | 6    | 10    |
+
+## Fighting pain
+
+Conscious soldiers can use new pain suppressing medication - APAP pills. These little bad boys with high dose of paracetamol are safe to use - they pose little to no chance of overdosing. The drawback is that they can't be used on unconscious units.
+
+| Feature                            | Apap                   | Morphine                                 |
+|------------------------------------|------------------------|------------------------------------------|
+| Max efficiency time                | 10 min                 | 30 min                                   |
+| Delay before kicking in            | 2 min                  | 0.5 min                                  |
+| Painkilling strength               | 0.6                    | 0.8                                      |
+| Use case                           | Conscious units in pain| Great pain and unconscious units in pain |
+| Side effects                       | None                   | Lowers pulse                             |
+| Chance of inducing unconsciousness | Negligible             | Very high, 2 injections might lead to loss of consciousness|
```

Comment:
```suggestion
| Feature                            | Apap                    | Morphine                                                    |
| ---------------------------------- | ----------------------- | ----------------------------------------------------------- |
| Max efficiency time                | 10 min                  | 30 min                                                      |
| Delay before kicking in            | 2 min                   | 0.5 min                                                     |
| Painkilling strength               | 0.6                     | 0.8                                                         |
| Use case                           | Conscious units in pain | Great pain and unconscious units in pain                    |
| Side effects                       | None                    | Lowers pulse                                                |
| Chance of inducing unconsciousness | Negligible              | Very high, 2 injections might lead to loss of consciousness |
```

Reply:
Welp, I should really put my interest towards these markdown VS plugins ;)